RELEASE/scripts/autoscend.ash

```diff
@@ -165,6 +167,7 @@ void initializeSettings() {
 	set_property("auto_drunken", "");
 	set_property("auto_eaten", "");
 	set_property("auto_familiarChoice", "");
+	set_property("auto_farmingKaAsEd", false);
```

Comment:
I suggest making the property name _auto_farmingKaAsEd so it self deletes when not in use